AirCasting/SDSync/ViewsFlow/BackendSyncCompleted/BackendSyncCompletedViewModel.swift

```diff
@@ -0,0 +1,40 @@
+// Created by Lunar on 02/12/2021.
+//
+
+import Foundation
+
+enum ProceedToSyncView {
+    case restart
+    case bluetooth
+}
+
+protocol BackendSyncCompletedViewModel: ObservableObject {
+    var presentRestartNextScreen: Bool { get set }
+    var presentBTNextScreen: Bool { get set }
+    var urlProvider: BaseURLProvider { get }
+    func procedToBTScreen()
+    func procedToRestartScreen()
```

Comment:
usually VM interfaces don't look like that in terms of naming. Views don't tell viewmodels what to do exactly, they only inform it on what happend on the UI. For example:
instead of:
```Swift
func continueToNextScreen()
```
we should use:
```Swift
func continueButtonTapped()
```
It shifts responsibility on how to handle actions to ViewModel which is its whole job.
(Hope my explanation is clear)
please try to change those functions here accordingly.